code/setup.dm

```diff
@@ -243,6 +243,21 @@ var/MAX_EXPLOSION_RANGE = 14
 #define SLOT_TWOEARS 8192
 #define SLOT_LEGS = 16384
 
+
+//MANNEQUIN SLOT BITMASKS
+#define SLOT_MANNEQUIN_ICLOTHING	"uniform"
+#define SLOT_MANNEQUIN_FEET			"shoes"
+#define SLOT_MANNEQUIN_GLOVES		"gloves"
+#define SLOT_MANNEQUIN_EARS			"earset"
+#define SLOT_MANNEQUIN_OCLOTHING	"suit"
+#define SLOT_MANNEQUIN_EYES			"glasses"
+#define SLOT_MANNEQUIN_BELT			"belt"
+#define SLOT_MANNEQUIN_MASK			"mask"
+#define SLOT_MANNEQUIN_HEAD			"hat"
+#define SLOT_MANNEQUIN_BACK			"backpack"
+#define SLOT_MANNEQUIN_ID			"idcard"
```

Comment:
As I feared it didn't work. Because of all that "lists and numbers and indexes" problem